docs/source/dataformat.rst

```diff
@@ -744,24 +793,181 @@ According to the table above, a song containing two verses (*v1, v2*), a chorus
+if not present, the default value is the ``major``. It can be marked
+
+- with a sorthand code, or
+- with a chord formula (for experts).
+
+These are the built-in **sorthand codes**:
+
+============ =================================== ========
+Shortcode    Chord Name                          Notation
+============ =================================== ========
+**power**    perfect 5th; power chord            5
+             major
+**min**      minor                               m
+**aug**      augmented                           \+
+**dim**      diminished                          °
+**dom7**     dominant 7th                        7
+**maj7**     major 7th                           Δ
+**min7**     minor 7th                           m7
+**dim7**     diminished 7th                      °7
+**halfdim7** half-diminished 7th                 ø
+**minmaj7**  minor major 7th                     mΔ
+**augmaj7**  augmented major 7th                 +Δ
+**aug7**     dominant 7th sharp 5; augmented 7th 7
+**maj6**     major 6th                           6
+**maj6b**    (major minor 6th)                   6♭
+**min6**     minor 6th                           m6
+**min6b**    (minor minor 6th)                   m6♭
+**dom9**     (dominant) 9th                      9
+**dom9b**    dominant minor 9th                  7,9♭
+**maj9**     major 9th                           Δ9
+**min9**     minor (dominant)Í 9th               m9
+**minmaj9**  minor major 9th                     mΔ9
+**aug9**     augmented (dominant) 9th            9
+**halfdim9** half-diminished 9th                 ø9
+**sus4**     suspended 4th                       4
+**sus2**     suspended 2nd                       2
+**add9**     original name + added 9th           add9
+============ =================================== ========
+
+Other chords can be noted with **chord formulas**. OpenLyrics has 69 built-in chords defined by a formula.
+Using chord formulas, an author can write additional custom chords. Chord formulas are described
+in :ref:`chord formulas <chordlist>`.
+
+To display root+structure+bass
+""""""""""""""""""""""""""""""
+
+The processors should display chords:
+
+- First display the ``root`` according to ``chordNotation``.
+- Immediately followed by the notation for the marked chord.
+- If there is a bass: immediately followed by a slash (/) and the ``root`` according to ``chordNotation``.
+
+Examples:
+
+=============================================== =========
+XML                                             Displayed
+=============================================== =========
+``<chord root="C" structure="dom7"/>``          C7
+``<chord root="D" bass="F#"/>``                 D/F♯
+``<chord root="C" structure="min" bass="Eb"/>`` Cm/E♭
+``<chord root="E" structure="3-5-m7-13"/>``     E7,6
+=============================================== =========
+
+Mixing lyrics and chords
+""""""""""""""""""""""""
+
+The ``<chord>`` tags are mixed in with the lyrics of a song::
 
```

Comment:
Previously, no text was allowed inside tag chord. What is the purpose of having the ability to have text inside chord tag? To alow following examples?

Reply:
This is a long story. Please see explanation is #54.

Reply:
Understood.